policy-packs/aws-ts-finops/ec2Compute.ts

```diff
@@ -0,0 +1,192 @@
+// Copyright 2016-2019, Pulumi Corporation.  All rights reserved.
```

Comment:
```suggestion
// Copyright 2016-2022, Pulumi Corporation.  All rights reserved.
```

If you cut/pasted from somewhere else, that place needs an update too ;)